src/python/grpcio/grpc/_cython/_cygrpc/security.pyx.pxi

```diff
@@ -14,14 +14,14 @@
 
 from libc.string cimport memcpy
 
-import pkg_resources
+import pkgutil
 
 
 cdef grpc_ssl_roots_override_result ssl_roots_override_callback(
     char **pem_root_certs) nogil:
   with gil:
-    temporary_pem_root_certs = pkg_resources.resource_string(
-        __name__.rstrip('.cygrpc'), '_credentials/roots.pem')
+    temporary_pem_root_certs = pkgutil.get_data(
+        __name__[:-len('.cygrpc')], '_credentials/roots.pem')
```

Comment:
Can we change back to `rstrip` here? These two is not always equivalent when the `__name__` not ending with `.cygrpc`.

Reply:
`rstrip` is the wrong method to use here; the argument is [not a suffix](https://docs.python.org/3/library/stdtypes.html#str.rstrip). My latest update changes the code to do what was probably originally intended, though.